pkg/extract/extract.go

```diff
@@ -10,19 +10,21 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/cheggaaa/pb/v3"
 	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/mattn/go-isatty"
 	"github.com/xi2/xz"
 )
 
-func UncompressWithFilter(tarball, targetDir string, fileFilter func(string) bool) ([]string, error) {
-	return uncompress(tarball, targetDir, fileFilter)
+func UncompressWithFilter(tarball, targetDir string, showProgress bool, fileFilter func(string) bool) ([]string, error) {
+	return uncompress(tarball, targetDir, fileFilter, showProgress)
 }
 
-func Uncompress(tarball, targetDir string) ([]string, error) {
-	return uncompress(tarball, targetDir, nil)
+func Uncompress(tarball, targetDir string, showProgress bool) ([]string, error) {
+	return uncompress(tarball, targetDir, nil, showProgress)
 }
 
-func uncompress(tarball, targetDir string, fileFilter func(string) bool) ([]string, error) {
+func uncompress(tarball, targetDir string, fileFilter func(string) bool, showProgress bool) ([]string, error) {
```

Comment:
where is the `showProgress` bool is used in this function, I think you need to pass it to the untarFile function and hide the progessbar as per this bool.

Reply:
fixed, thanks!